### ISSUE TITLE
Link underlines

### ISSUE THREAD

**Issue by ppvg** [minvws/nl-rdo-manon#423]
From independent accessibility report, SC 1.4.1:

<blockquote lang="nl">
Het volgende is geen fout binnen een component, maar wel binnen de aangeleverde instructies: Op verschillende pagina's staan links in de lopende tekst. Op pagina <a href="https://minvws.github.io/nl-rdo-manon/documentation/components/button.html">components/button.html</a> gaat het bijvoorbeeld om de links "Beschikbare types" en "Beschikbare button types". Dat dit links zijn, wordt nu enkel overgebracht doordat ze een andere kleur hebben. Dit is niet voor iedereen goed zichtbaar. Het advies is om links in een lopende tekst altijd te onderstrepen. Eventueel mag het contrast tussen de linktekst en de omliggende tekst ook verhoogd worden tot minstens 3,0:1.
</blockquote>

The documentation theme (`soft`) is missing underlines on links.

**Comment by HeleenSG** [comment]
Theme soft is deprecated. As such this issue is no longer relevant.

**Comment by HeleenSG** [comment]
Solved in new version of Manon on main (feat/manon-sass)
